Bottom: 7ad112cbef165c8f26fd9199dcd77a5ffa88c36e
Top:    a2411eddc51b22f03ceca94eda25a1f1f44d2090
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-22 05:01:46 +0100

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
